pkgs/by-name/tr/try/package.nix

```
@@ -1,37 +1,68 @@
 {
   stdenvNoCC,
+  autoreconfHook,
   lib,
   fetchFromGitHub,
-  fuse-overlayfs,
   util-linux,
+  mergerfs,
+  attr,
   makeWrapper,
+  pandoc,
+  gcc9,
+  kmod,
+  installShellFiles,
 }:
 stdenvNoCC.mkDerivation rec {
   pname = "try";
-  version = "0.2.0";
+  version = "20250515-git";
   src = fetchFromGitHub {
     owner = "binpash";
     repo = "try";
-    rev = "v${version}";
-    hash = "sha256-2EDRVwW4XzQhd7rAM2rDuR94Fkaq4pH5RTooFEBBh5g=";
+    rev = "67052d8f20725f3cdc22ffaec33f7b7c14f1eb6b";
+    hash = "sha256-8mfCmqN50pRAeNTJUlRVrRQulWon4b2OL4Ug/ygBhB0=";
   };
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [
+    makeWrapper
+    autoreconfHook
+    pandoc
+    gcc9
+    pandoc
+    attr
+    util-linux
+    kmod
+    installShellFiles
+  ];
+  # Trigger a overlay run to load the overlayfs module, required by ./configure
+  preConfigure = ''
+    mkdir lower upper work merged
+    unshare -r --mount mount -t overlay overlay -o lowerdir=lower,upperdir=upper,workdir=work merged
+    rm -rf lower upper work merged
+  '';
   installPhase = ''
     runHook preInstall
     install -Dt $out/bin try
+    install -Dt $out/bin utils/try-commit
+    install -Dt $out/bin utils/try-summary
     wrapProgram $out/bin/try --prefix PATH : ${
       lib.makeBinPath [
-        fuse-overlayfs
         util-linux
+        mergerfs
+        attr
       ]
     }
     runHook postInstall
   '';
+  postInstall = "installManPage man/try.1.gz";
```

nit: but I would install the manpages with the binary too.
```suggestion
        attr
      ]
    }
    installManPage man/try.1.gz
    runHook postInstall
  '';
```